src/httpParsing/parseHar.ts

```diff
@@ -11,7 +11,27 @@ const getEntries = (harFile: Record<string, any>) => {
   return entries;
 };
 
+const parseContent = (input: {
```

Comment:
Nit-picking perhaps, but this method doesn't actually parse content. It seems more like tryFormContenNode or something.